gpu_bdb/benchmark_runner/slurm/rapids-gpu-bdb-cuda11.yml

```diff
@@ -24,6 +24,9 @@ dependencies:
   - jupyterlab
   - gspread
   - oauth2client
+  - numba=0.54.*
```

Comment:
do we have a reason to maintain 11.0 anymore?

Reply:
No reason to maintain the 11.0 docker file. Also related, I'll update the Numba constraint to >=0.54 since we have 0.55 as well